.github/workflows/docs-caller.yml

```
@@ -0,0 +1,106 @@
+      docstring-threshold: 80
+      deploy-to-pages: true  # Deploy on main branch
+
+  # Documentation summary
+  docs-summary:
+    name: Documentation Summary
+    needs: [validate-frontmatter, build]
+    runs-on: ubuntu-latest
+    if: always()
+    steps:
+      - name: Generate summary
+        run: |
```

actionlint (1.7.10)

shellcheck reported issue in this script: SC2086:info:11:45: Double quote to prevent globbing and word splitting

(shellcheck)